src/test/java/edu/hawaii/its/api/controller/GroupingsRestControllerv2_1Test.java

```diff
@@ -404,50 +405,50 @@ public void memberAttributeResultsAsyncTest() throws Exception {
         verify(memberAttributeService, times(1))
                 .getMemberAttributeResultsAsync(UID, members);
     }
-
     @Test
     public void ownedGroupingTest() throws Exception {
+        SortBy[] sortByOptions = { SortBy.NAME, SortBy.UID, SortBy.UH_UUID };
         String json = propertyLocator.find("ws.get.members.results.success.multiple.groups");
         WsGetMembersResults wsGetMembersResults = JsonUtil.asObject(json, WsGetMembersResults.class);
         GetMembersResults getMembersResults = new GetMembersResults(wsGetMembersResults);
         GroupingGroupsMembers groupingGroupsMembers = new GroupingGroupsMembers(getMembersResults);
-        assertNotNull(groupingGroupsMembers);
-        List<String> paths =
-                Arrays.asList("group-path:basis", "group-path:include", "group-path:exclude", "group-path:owners");
-        given(groupingOwnerService.paginatedGrouping(CURRENT_USER, paths, 1, 700, "name", true))
+        List<String> paths = Arrays.asList("group-path:basis", "group-path:include", "group-path:exclude", "group-path:owners");
+        for (SortBy sortBy: sortByOptions) {
+            given(groupingOwnerService.paginatedGrouping(CURRENT_USER, paths, 1, 700, sortBy.sortString(), true))
```

Comment:
This given().willReturn() can be put above the for loop because it only needs to be set once.